src/fides/api/ops/service/connectors/fides_connector.py

```diff
@@ -182,6 +182,6 @@ def filter_fides_connector_datasets(
     return {
         dataset.fides_key
         for connector_config in connector_configs
-        for dataset in connector_config.datasets
+        for dataset in connector_config.datasets  # type: ignore[attr-defined]
```

Comment:
ahhh why are these connection config attributes suddenly causing problems

Reply:
idk 😢 but it was a lot